File: b_rank_skillcheck_archive/3d_printer.py
# 実行時間 20ms
import sys

r = sys.stdin.readline
x, y, z = list(map(int, r().split()))
o = []

for i in range(z):
    t = ["." for _ in [0] * y]

    for _ in [0] * x:
        u = r().strip()

        for j in range(y):
            if u[j] == "#":
                t[j] = "#"
    o.append("".join(t))
    r()
o.reverse()
print("\n".join(o))

File: b_rank_skillcheck_archive/3d_printer.cs
// 実行時間 20ms
using static System.Console;
using static System.Linq.Enumerable;

internal class Program
{
    static void Main()
    {
        int[] xyz = ReadLine().Split().Select( int.Parse ).ToArray();
        string[] o = new string[xyz[2]];

        foreach( int i in Range( 0, xyz[2] ) ) {
            char[] t = Repeat( '.', xyz[1] ).ToArray();

            foreach( int _ in Range( 0, xyz[0] ) ) {
                string r = ReadLine();

                foreach( int j in Range( 0, xyz[1] ) ) {
                    if( r[j] == '#' ) {
                        t[j] = '#';
                    }
                }
            }
            o[i] = string.Join( "", t );
            ReadLine();
        }
        WriteLine( string.Join( System.Environment.NewLine, o.Reverse() ) );
        return;
    }
}
